pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -16,34 +15,28 @@
 , darwin
 , elfutils # for libdw
 , bash-completion
-, docbook_xsl
-, docbook_xml_dtd_43
-, gtk-doc
 , lib
 , CoreServices
 }:
 
 stdenv.mkDerivation rec {
   pname = "gstreamer";
-  version = "1.16.2";
+  version = "1.18.0";
 
-  outputs = [ "out" "dev" "devdoc" ];
+  outputs = [
+    "out"
+    "dev"
+    # "devdoc" # disabled until `hotdoc` is packaged in nixpkgs
```

Comment:
```suggestion
    # "devdoc" # disabled until `hotdoc` is packaged in nixpkgs, see:
    # - https://github.com/NixOS/nixpkgs/pull/98767
    # - https://github.com/NixOS/nixpkgs/issues/98769#issuecomment-702296551
```

Reply:
I wonder if it even makes sense to try to build docs ourselves instead of using a separate package just extracting the tarball, as suggested in the release notes https://github.com/NixOS/nixpkgs/pull/99345#issuecomment-714186302